src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -640,10 +641,32 @@ public AdsBaseObject addToUrl(String param) {
     return this;
   }
 
-  public void checkSkin(String adSkinLeftPath, String adSkinRightPath) {
-    AdSkinHelper skinHelper = new AdSkinHelper(adSkinLeftPath, adSkinRightPath, driver);
+  public void checkSkin(String adSkinLeftPath,
+                        String adSkinRightPath,
+                        String backgroundColor,
+                        String middleColor) {
+    SkinHelper skinHelper = new SkinHelper(adSkinLeftPath, adSkinRightPath, driver);
     Assertion.assertTrue(skinHelper.skinPresent());
     PageObjectLogging.log("SKIN", "SKIN presents on the page", true);
+    if (backgroundColor != null) {
```

Comment:
I don't think there's any reason to check whether the creative has the expected size in it